Update takeoperator.md (#181)
MicrosoftDocs/dataexplorer-docs: data-explorer/kusto/query/takeoperator.md

when browsing data interactively, but be aware that it doesn't guarantee any consistency
in its results when executing multiple times, even if the data set hasn't changed.

> Even if the number of rows returned by the query is not explicitly limited

Reviewer: ```suggestion
Even if the number of rows returned by the query isn't explicitly limited
```
